Penteract/Assets/Scripts/RangedAI.cpp

```diff
@@ -607,6 +630,7 @@ void RangedAI::PlayAudio(AudioType audioType) {
 void RangedAI::EnableBlastPushBack() {
 	if (state != AIState::START && state != AIState::SPAWN && state != AIState::DEATH) {
 		ChangeState(AIState::PUSHED);
+		particlesPush->PlayChildParticles();

```

Comment:
Check nullity. In the start you check the nullity of the aux local variable, but at this point this pointer could be null